.gitignore

```diff
@@ -0,0 +1,3 @@
+# Ignoring compiled python modules
+*.pyc
+
```

Comment:
This is useless if you are using Python 3 as you should

Reply:
For .gitignore you can consider the templates available here https://github.com/github/gitignore

Reply:
I will move to Py3 soon. For the moment I'll leave the gitignore as it is (not arming, I guess)

Reply:
+1 for @mmpagani